tests/components/bmw_connected_drive/snapshots/test_sensor.ambr

```diff
@@ -85,6 +88,7 @@
     StateSnapshot({
       'attributes': ReadOnlyDict({
         'attribution': 'Data provided by MyBMW',
+        'device_class': 'distance',
```

Comment:
I would expect it would look like this

```suggestion
        'device_class': '<Sensor device class=distance,>',
```

Reply:
This seems to be coming up only when iterating over config entries as done e.g. in the [`airly` tests](https://github.com/home-assistant/core/blob/dev/tests/components/airly/test_sensor.py#L32).

I don't think it makes a difference.